docs/README.md

```diff
@@ -1,22 +1,45 @@
-# go.bytecodealliance.org Go import paths
+# Go Import Paths
 
-This directory hosts the configuration for Go vanity URLs under `go.bytecodealliance.org` domain.
+This directory hosts the configuration for [Go module import paths](https://golang.org/cmd/go/#hdr-Remote_import_paths) under the `go.bytecodealliance.org` domain. Each unique Go module (with a separate `go.mod` file) must have a corresponding directory and `index.html` file linking to this repository (or another respository, see below).
 
-More information about vanity URLs can be found in the [Go documentation](https://golang.org/cmd/go/#hdr-Remote_import_paths).
+Packages within a named module, e.g. `go.bytecodealliance.org/cm` will automatically be found.
 
-## Adding a new vanity URL
+## Creating a New Module
 
-If you want to create a vanity URL `go.bytecodealliance.org/foo` pointing to `github.com/bytecodealliance/go-foo`:
+### In This Repository
 
-- Create directory `docs/foo`
-- Create file `docs/foo/index.html` with the following content:
+To create a new Go module `go.bytecodealliance.org/gopher` **in this repository**:
 
-     ```html
-     <html>
-     	<head>
-     		<meta name="go-import" content="go.bytecodealliance.org/foo git https://github.com/bytecodealliance/go-foo" />
-     	</head>
-     </html>
-     ```
+1. Create a new `gopher` directory in the root of this repository.
+2. In the `gopher` directory, run `go mod init go.bytecodealliance.org/gopher`.
+3. Create a new `docs/gopher` directory.
+4. Create a file `docs/gopher/index.html` with the following content:
 
-NOTE: Vanity URLs must be unique and not shadow existing package names in this repository. For example: Creating a vanity URL `go.bytecodealliance.org/cm` would shadow the `cm` package in this repository.
+	```html
+	<html>
+		<head>
+			<meta name="go-import" content="go.bytecodealliance.org/gopher git https://github.com/bytecodealliance/go-modules" />
+		</head>
+	</html>
+	```
+5. Commit these changes and submit a Pull Request to this repository.
+
+### In Another Repository
```

Comment:
Can you add some guidance on how to migrate existing repo as a new module (e.g. wasmtime-go)?